test/e2e/framework.go

```diff
@@ -120,7 +120,7 @@ const (
 	antreaControllerConfName = "antrea-controller.conf"
 	flowAggregatorConfName   = "flow-aggregator.conf"
 
-	agnhostImage        = "registry.k8s.io/e2e-test-images/agnhost:2.40"
+	agnhostImage        = "registry.k8s.io/e2e-test-images/agnhost:2.52"
 	ToolboxImage        = "antrea/toolbox:1.3-0"
```

Comment:
This should also be updated to 'toolbox:1.4-0'.
Please check if there are other occurrences.